com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/resources/jsweb/dmloader.js

```diff
@@ -133,6 +133,31 @@ var EngineLoader = {
             });
     },
 
+    streamAndInstantiateWasmAsync: function(src, fromProgress, toProgress, callback) {
+        Module.instantiateWasm = function(imports, successCallback) {
+            WebAssembly.instantiateStreaming(fetch(src), imports).then(function(output) {
+                Progress.calculateProgress(fromProgress, toProgress, 1, 1);
```

Comment:
It is not possible to get the progress from WebAssembly.instantiateStreaming(). It can be done through a custom Fetch Response. Is it worth it?

Reply:
No, I don't think so.